Python/bytecodes.c

```diff
@@ -3006,6 +3006,7 @@ dummy_func(
             tstate->py_recursion_remaining--;
             LOAD_SP();
             LOAD_IP();
+            frame->prev_instr = _PyCode_CODE(_PyFrame_GetCode(frame));
```

Comment:
See my comment in ceval_macros.c -- perhaps this should also be a macro?

Reply:
This is unnecessary, `frame->prev_instr` is set during frame creation.

Reply:
And if we want to reuse this code for `SEND` we don't want to be messing with `frame->prev_instr`

Reply:
> This is unnecessary, frame->prev_instr is set during frame creation.

It's set to an index of -1 (since the frame hasn't started yet), which is invalid. So we either need to set it like this, or at least do an increment.